lib/fluent/plugin/filter_process_snmptrap.rb

```diff
@@ -80,12 +94,12 @@ def processEvent(record)
       end
     end
 
-    def getrmchost(record)
-      rmc_host = record[@@host]
-      if rmc_host.nil?
+    def getrmcHost(record)
```

Comment:
do we really need this? It seems we are just reassigning host to rmcHostIP. Can't we just rely on host?